bucket/prismlauncher-qt5.json

```diff
@@ -3,12 +3,27 @@
     "description": "An open source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
-    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.3/PrismLauncher-Windows-MSVC-Legacy-6.3.zip",
-    "hash": "003b54560935c341ab825d41962323e56d649ea15a29f69d4ce64b93326c866c",
+    "notes": [
```

Comment:
Maybe add a note about the migration not deleting the old AppData directory? As well as maybe making the notes more concise:
```
        "Run this for file associations with .ZIPs and .MRPACKS: \"$dir\\install-associations.reg\"",
        "This package is now fully portable, with data being the in the Scoop persist directory, and it should have been migrated automatically. If this is a global Scoop install, you will have to manually copy your data from AppData to persist."
```